src/executeAutomagically.ts

```diff
@@ -18,6 +18,15 @@ const sleep = (timeInMilliseconds: number): Promise<void> =>
 const getExecuteUrl = (automagicallyUrl: string) =>
   `${automagicallyUrl}/api/apiKey/v2/execute`
 
+const multilineMappingToObject = (input: string[]): Record<string, string> => {
+  const keySplitOff = input
+    .filter(mapping => mapping.length > 0)
+    .map(mapping => mapping.split(':'))
+    // the api takes an array of values per key, so we just wrap the value in an array
+    .map(parts => [parts[0], [parts.slice(1).join(':')]])
```

Comment:
actually the join(':') confused me, maybe we can add to the comment that we are restoring the ':' in the value as well